api/tests/integration/application/scenarios-simulator/scenario-simulator-controller_test.js

```diff
@@ -343,7 +334,7 @@ describe('Integration | Application | scenario-simulator-controller', function (
             ];
 
             const pickChallengeImplementation = sinon.stub();
-            pickChallengeService.chooseNextChallenge.withArgs(`${assessmentId}-0`).returns(pickChallengeImplementation);
+            pickChallengeService.chooseNextChallenge.returns(pickChallengeImplementation);
```

Comment:
Est-ce qu'on s'attends bien a ne pas avoir de parametre?
```suggestion
pickChallengeService.chooseNextChallenge.withArgs().returns(pickChallengeImplementation);
```